File: GFG/Easy/0_StringDuplicatesRemoval.py
"""
Problem Link : https://practice.geeksforgeeks.org/problems/remove-all-duplicates-from-a-given-string4321/1
Platform     : GFG
Difficulty   : Easy
"""

class Solution:

	
	def removeDuplicates(self, s):
	    dict ={}
	    for i in s:
	        if i in dict.keys():
	            dict[i] = dict[i]+1;
	        else:
	           dict[i] =1;
	    string = ""
	    for i in dict.keys():
	        
	        
	        string= string+i
	   
	    return string

File: GFG/Easy/0_StringDuplicatesRemoval.java
/**
 * Problem Link : https://practice.geeksforgeeks.org/problems/remove-all-duplicates-from-a-given-string4321/1
 * Platform     : GFG
 * Difficulty   : Easy
 */

class Solution {
    String removeDuplicates(String s) {
        StringBuilder sb = new StringBuilder();
        for( int i =0; i<s.length(); i++){
            char ch = s.charAt(i);
            if(sb.indexOf(String.valueOf(ch)) ==-1){
                sb.append(ch);
            }
            
        }
        String result = sb.toString();
        return result;
       
        }
}

